app/operations/shopify/product_operations.rb

```diff
@@ -0,0 +1,27 @@
+module Shopify
+  class ProductOperations
```

Comment:
Does this class name make sense? Does it express what it's doing without the use of one of it's class methods? If some developer was code-splunking in the host app would they be clued into what's going on or at least be able to fill in the gaps?